### DISCUSSION TITLE
글목록에서 구분선이 표시되지 않습니다

### DISCUSSION THREAD

**Discussion by JongHoooon** [pronist/hELLO#140] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: macOS Monterey
- 브라우저: chrome
- 버전: 106.0.5249.119
- 스킨 버전:  3.6.3

**설명**
목차에서 구분선이 표시되지 않고 글문단이 다른분들보다 크게 나타납니다.

**발생 과정**
글목록들이 구분선도 없고 크기도 크게 나옵니다.

**스크린샷 및 링크**

https://bromp.tistory.com/category

<img width="1179" alt="image" src="https://user-images.githubusercontent.com/98168685/198267035-237f2f75-b6c1-44f2-963c-9ab68a214377.png">

**Comment by pronist** [comment, marked as answer]
@seaLion3 안녕하세요.

스킨 커버 및 스킨 옵션에서 글목록 스타일을 변경하지 않으신 것 같으니 확인해보시기 바랍니다.
